src/odemis/acq/_futures.py

```diff
@@ -124,9 +124,9 @@ def _run(self):
         self._stream.is_active.value = True
 
         # wait until one image acquired or cancelled
-        if not self._acq_over.wait(10 * estt + 5):
```

Comment:
This a hack that is caused because the stream now take care of calling the OPM by themselves. Just call prepare() explicitly, and revert to the previous time-outs.